app/rest/registry/blobs.js

```diff
@@ -0,0 +1,16 @@
+angular.module('portainer.rest')
+.factory('RegistryBlobs', ['$resource', 'API_ENDPOINT_REGISTRIES', function RegistryBlobsFactory($resource, API_ENDPOINT_REGISTRIES) {
+  'use strict';
+  return $resource(API_ENDPOINT_REGISTRIES + '/:id/v2/:repository/blobs/:reference', {}, {
+    head: {
+      method: 'HEAD',
+      params: { id: '@id', repository: '@repository', reference: '@reference' },
+      transformResponse: function(data, headers){
```

Comment:
Why is this required?

Reply:
We need to get the response headers here
We don't download layers, so we send HEAD request, and get content-length header
See https://github.com/portainer/portainer/pull/1482/files#diff-7b71727483ce8ff9ef617853d8964592R53

Reply:
OK